core/src/main/java/io/lionweb/lioncore/java/self/LionCore.java

```diff
@@ -164,9 +164,13 @@ private LionCore() {
 
   public static @Nonnull Language getInstance(@Nonnull LionWebVersion lionWebVersion) {
     Objects.requireNonNull(lionWebVersion, "lionWebVersion should not be null");
+    String versionIDSuffix = "";
+    if (lionWebVersion != LionWebVersion.v2023_1) {
+      versionIDSuffix = "-" + lionWebVersion.getVersionString().replaceAll("\\.", "-");
```

Comment:
Also feels like some magical string adjustment that's implemented in several places. Consolidate?

Reply:
I created a method in IdUtils to contain this logic